config.sample

```diff
@@ -1,65 +1,107 @@
 
 # Twilio config
 $ApiVersion = "2010-04-01";                          # Do not change
-$AccountSid = "AC01234567890123456789012345678901";  # Get these from https://www.twilio.com/console
-$AuthToken = "01234567890123456789012345678901";
+#TODO:: assign $AccountSid, ~20 character string from https://www.twilio.com/console
+$AccountSid = "";                             # ***NOT*** phone number, get from https://www.twilio.com/console
+#TODO:: assign $AuthToken, ~20 character string from https://www.twilio.com/console, Twilio's equivalent of a password
+$AuthToken = "";                                # Get from https://www.twilio.com/console
 
 # Database config
-$db_host = "dbhostname.sample.com";
-$db_name = "databasename";
-$db_user = "dbuser";
-$db_pass = "dbpass";
-$table_name = "opencrisisline"; 
-
-# Additional menu items, comment out or set to NULL to disable them 
-$option2_column = "opt2";                      
+#TODO:: assign $db_host the address of the database, e.g., localhost, IP address, fully qualified domain name
+$db_host = "";
+#TODO:: assign $db_name the name of the database containing volunteer phone numbers and handles, e.g., "ocl_db"
+$db_name = "";
+#TODO:: assign $db_user an authorized user of the above database, e.g., "adminperson"
+$db_user = "";
+#TODO:: assign $db_pass the password for $db_user to access the $db_name database, e.g., "AwfulPassword2!"
+$db_pass = "";
+#TODO:: assign $db_charset the character set used in the $db_name database, e.g., "utf8mb4";
+$db_charset = "";
+#TODO:: assign $table_name the table to maintain the volunteers' phone number, handle, and responsibilities
+$table_name = "";
+
+# Additional menu items, comment out or set to NULL to disable them
+#TODO:: assign $option2_column the column name for a second volunteer responsibility, e.g., "opt2"
+$option2_column = "opt2";
+#TODO:: assign $option2_friendly name for $option2_column that is used for volunteer responsibilities registration and in the caller's menu
 $option2_friendly = "Code Red";
+#TODO:: assign $option2_column the column name for a third volunteer responsibility, e.g., "opt3"
 $option3_column = "opt3";
+#TODO:: assign $option3_friendly name for $option3_column that is used for volunteer responsibilities registration and in the caller's menu
+#TODO:: $option3_friendly must contain the word "Graveyard" for graveyard calling (call less people) to be enacted
 $option3_friendly = "Graveyard / Night Shift";
 
 require "twilio.php";
 
 function logError($err) {
-    global $admin_email;
-    mail($admin_email,"[$system_name] Error","http://".$_SERVER["SERVER_NAME"].$_SERVER["REQUEST_URI"]."\n\n".print_r($_REQUEST,true)."\n\n\n".print_r($err,true));
+    global $system_name, $admin_email;
+#TODO::    mail($admin_email,"[$system_name] Error","http://".$_SERVER["SERVER_NAME"].$_SERVER["REQUEST_URI"]."\n\n".print_r($_REQUEST,true)."\n\n\n".print_r($err,true));
```

Comment:
Should this be disabled by default?  Or is it better to have mail() call in there by default, and make a new config system to skip it if one needs?

Reply:
No, should not be disabled.

As you probably picked up, the #TODO:: is a flag used by PhpStorm so one can track work that needs to be done. And easy to grep on if not using PhpStorm. In this particular case, I meant to go back and uncomment it out. Didn't. Good catch.